storage/sql/migrations/updates/20190204170311_add_asset_field_as_json_to_trs_table.sql

```diff
@@ -12,4 +12,13 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-INSERT INTO "transfer" ("transactionId", "data") VALUES ${values:raw};
+
+/*
+  DESCRIPTION: Add asset Column to trs.
+
+  PARAMETERS: None
+*/
+
+ALTER TABLE "trs" ADD COLUMN IF NOT EXISTS "asset" jsonb;
+
+CREATE INDEX IF NOT EXISTS "trs_asset" ON "trs" USING gin ("asset");
```

Comment:
Please communicate to `devops` team, maintaining `GIN` index can be quicker if we increase this configuration option in our PotgreSQL builds. 

https://www.postgresql.org/docs/9.1/runtime-config-resource.html#GUC-MAINTENANCE-WORK-MEM